plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2441,3 +2441,26 @@ function wc_update_670_purge_comments_count_cache() {
 
 	WC_Comments::delete_comments_count_cache();
 }
+
+/**
+ * Create feature toggle options for the legacy Admin feature toggles (see the FeaturesController class).
+ */
+function wc_update_700_migrate_legacy_feature_toggles() {
```

Comment:
Can we get rid of these migrations by using these custom names as fallback (maybe via a filter)? The use case I am concerned about is someone having a feature enabled, and then when they update to WC 7.0, it will be disabled till the time migrations are run.

Perhaps we can add a filter in `feature_is_enabled` or in `feature_enable_option_name` to support old names. Alternatively, we can do a live migration as well by hooking into the old option name ourselves rather than WC migration to complete.

Reply:
As discussed, the old feature names are used instead for these two legacy features.